### ISSUE TITLE
App keeps running even after closing fully...
And i have to force stop it from settings 

### ISSUE THREAD

**Issue by U1M450W** [OxygenCobalt/Auxio#271]

**Comment by OxygenCobalt** [comment]
There should be an X button in the notification to stop the app from running in the background.

**Comment by U1M450W** [comment]
Yes there's but.... it's just removing notification....but app is still working in background

**Comment by U1M450W** [comment]
> How so? It should stop music playback.

Music stops....but app doesn't stop and work in background untill i force close it

**Comment by U1M450W** [comment]
> > How so? It should stop music playback.
> 
> Music stops....but app doesn't stop and work in background untill i force close it

https://user-images.githubusercontent.com/116655729/197801347-5ddfe088-7e02-466c-b0d6-659f20161002.mp4

**Comment by U1M450W** [comment]
> That's strange. I'm doing everything right on my end, and I can't seem to reproduce it on my device (Pixel 5, Android 13), so I'm going to assume that this is a device-specific issue with the task manager on your device.

Maybe....but i tried another music app but... Didn't face this issues that time....

**Comment by OxygenCobalt** [comment]
Yeah, that's not happening for me. It's definitely device-specific.

https://user-images.githubusercontent.com/65370175/197803057-a565b145-e4cd-4d65-8a2e-a82810670614.mp4

Perhaps your system's task manager counts all running services as "active apps" and not just apps with a notification? Auxio's playback service does stick around for a little longer after the notification is closed.

**Comment by U1M450W** [comment]
> Yeah, that's not happening for me. It's definitely device-specific.
> 
> https://user-images.githubusercontent.com/65370175/197803057-a565b145-e4cd-4d65-8a2e-a82810670614.mp4
> 
> Perhaps your system's task manager counts all running services as "active apps" and not just apps with a notification? Auxio's playback service does stick around for a little longer after the notification is closed. The notification disappeared, so that must mean the service exited it's foreground state.

Ohh maybe.... because it's custom rom....so app doesn't drain battery if you close it with X button ?

**Comment by OxygenCobalt** [comment]
More or less. Even if the service is still alive, it's mostly idle and it's power usage is restricted by the system. Very unlikely that it's going to drain your battery.

**Comment by U1M450W** [comment]
> More or less. Even if the service is still alive, it's mostly idle and it's power usage is restricted by the system. Very unlikely that it's going to drain your battery. 

Cool then...thank you dev... For making such a good app 😀

And one more request....

Please add song deleting option 😅
Or i couldn't find it or it's not there

**Comment by OxygenCobalt** [comment]
Song deletion isn't in scope right now since newer android versions are exceedingly difficult. See #160.

**Comment by U1M450W** [comment]
> Song deletion isn't in scope right now since newer android versions are exceedingly difficult. See #160.

Ohh okay then...
